blockly-rtc/test/marker_update_test.js

```diff
@@ -0,0 +1,130 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Unit tests for MarkerUpdate.
+ * @author navil@google.com (Navil Perez)
+ */
+
+const assert = require('assert');
+const Blockly = require('blockly/dist');
+const sinon = require('sinon');
+
+const MarkerUpdate = require('../src/MarkerUpdate').default;
+
+suite('MarkerUpdate', () => {
+
+  suite('fromEvent()', () => {
+    setup(() => {
+      Blockly.defineBlocksWithJsonArray([{
+        'type': 'test_block',
+        'message0': 'test block'
+      }]);
+      this.FAKE_WORKSPACE_ID = 'mockWorkspaceId';
+      this.FAKE_BLOCK_ID = 'mockBlockId'
+      sinon.stub(Blockly.utils, 'genUid')
+      .onFirstCall().returns(this.FAKE_WORKSPACE_ID)
+      .onSecondCall().returns(this.FAKE_BLOCK_ID);
+      this.workspace = new Blockly.Workspace();
+      this.block = new Blockly.Block(this.workspace, 'test_block');
+    });
+
+    teardown(() => {
+      sinon.restore();
+      delete Blockly.Blocks['simple_test_block1'];
```

Comment:
The block type beign deleted doesn't match the type created in setup.

Reply:
Done.